src/buildQueryURL.ts

```diff
@@ -1,6 +1,17 @@
 import { castArray } from "./lib/castArray"
 import { devMsg } from "./lib/devMsg"
 
+/**
+ * The query parameter used to indicate if the client is in development mode to
+ * the API.
+ */
+export const PRISMIC_DEV_PARAM = "x-d"
+
+/**
+ * The query parameter used to indicate the version of the client to the API.
+ */
+export const PRISMIC_CLIENT_VERSION_PARAM = "x-c"
```

Comment:
❓ #ask: Should we make this strategy compatible with other languages? With the current version, analytics around `x-c` assume `@prismicio/client` is being used.

What happens if we want to track usage of other clients, like the PHP client? We only officially support the JavaScript ecosystem, but we may want to track usage of the languages to better understand how often other languages are used.

We may want to consider a URL scheme that works across languages.

See Imgix's `ix-lib` parameter for an example: https://github.com/imgix/js-core/blob/bfca36f5cac1833ff2ff390de4e42a15b215a89c/src/index.js#L114

Reply:
OK, what about `js@vX.Y.Z` as a syntax?